src/module/vue/components/legacy-track.vue

```diff
@@ -1,106 +1,255 @@
+  @max_track_width: (@max_progress_box_width*10 + @progress_box_gap*9);
+  .progress-track {
+    margin: 0;
+    gap: @progress_box_gap;
+  }
+  .progress-track-box {
+    // extra padding to allow comfy overlap with xp pips
+    padding-bottom: (@max_xp_box_width * 0.4);
+    max-height: unset;
+    max-width: @max_progress_box_width;
+    gap: @progress_box_gap;
+  }
+  .legacy-track-xp {
+    @xp_border_width: 1px;
+    grid-column: 1 / span 2;
+    grid-row: 3 / span 2;
+    display: grid;
+    grid-template-columns: repeat(20, 1fr);
+    max-width: @max_track_width;
+    width: 100%;
+    gap: @progress_box_gap;
+    justify-self: center;
+    position: relative;
+    .xp-box {
+      background-color: var(--ironsworn-color-bg);
+      margin: 0;
+      aspect-ratio: 1;
+      border-radius: 3px;
+      border-width: @xp_border_width;
+      width: 100%;
+      max-width: @max_xp_box_width;
+      z-index: 1;
+      &.hover,
+      &.selected {
+        background-color: var(--ironsworn-color-thematic);
+        z-index: 100;
+      }
+
+      &:not(:nth-child(n + 21)) {
+        &:nth-child(2n) {
+          justify-self: left;
+          border-top-left-radius: 0;
+          border-bottom-left-radius: 0;
+          margin-left: -((@progress_box_gap+ @xp_border_width)/2);
+        }
+        &:nth-child(2n + 1) {
+          justify-self: right;
+          border-top-right-radius: 0;
+          border-bottom-right-radius: 0;
+          margin-right: -((@progress_box_gap+ @xp_border_width)/2);
+        }
+      }
+      &:nth-child(n + 21) {
+        grid-column: span 2;
+        justify-self: center;
+      }
+    }
+  }
+}
+</style>
 <script setup lang="ts">
-import { computed, defineComponent, inject, Ref } from 'vue'
-import { IronswornActor } from '../../actor/actor'
+import { computed, inject, provide } from 'vue'
 import { $ActorKey } from '../provisions'
 import BtnFaicon from './buttons/btn-faicon.vue'
+import { capitalize } from 'lodash'
+import { IronswornActor } from '../../actor/actor.js'
 import XpTrack from './xp-track.vue'
+import _ from 'lodash'
+
 import ProgressTrack from './progress/progress-track.vue'
+import { CharacterDataProperties } from '../../actor/actortypes.js'
+import { ActorData } from '@league-of-foundry-developers/foundry-vtt-types/src/foundry/common/data/module.mjs.js'
+import { DocumentData } from '@league-of-foundry-developers/foundry-vtt-types/src/foundry/common/abstract/module.mjs.js'
+
+// TODO: make this use an enum from dataforged instead, once rsek gets around to adding it
+type LegacyType = 'quests' | 'bonds' | 'discoveries'
+
+// TODO: switch to Dataforged consts when available thru DF2
+const maxBoxes = 10
+const maxScore = maxBoxes
+const ticksPerBox = 4
+const maxTicks = maxBoxes * ticksPerBox
+const minTicks = 0
+const xpEarnedPerBox = 2
+const xpEarnedPerOverflowBox = 1
 
-const props = defineProps<{ propKey: string; title: string }>()
+const props = defineProps<{
+  actor: IronswornActor
```

Comment:
Same comment as before: you're injecting `$ActorKey`, so this node already has an actor. No need to pass it in as a prop, or re-provide it to nodes below.